content/en/getting_started/synthetics/_index.md

```diff
@@ -0,0 +1,215 @@
+
+To configure an API test:
+
+1. In the Datadog application, hover over **UX Monitoring** in the left hand menu and select **Synthetics Test**. 
+2. In the top right corner, click the **New Test** button. 
+3. Select **API test**.
+4. Define the configuration of your API test.
+
+{{< tabs >}}
+
+{{% tab "HTTP Test" %}}
+
+{{< img src="synthetics/api_tests/make-http-request.png" alt="Make HTTP Request" responsive="true" style="width:80%;" >}}
+
+Define the request you want to be executed by Datadog:
+
+1. **Choose request type**: `HTTP`
+2. Choose the **Method** and **URL** to query. Available methods are: `GET`, `POST`, `PATCH`, `PUT`, `HEAD`, `DELETE`, and `OPTIONS`.
+3. **Name**: Set the name of your API test.
+4. **Select your tags**: Select the tags attached to your API test. Use the `<KEY>:<VALUE>` format to filter on a `<VALUE>` for a given `<KEY>` on the [Synthetics page][1].
+5. **Locations**: Select which Datadog managed locations to run the test from. Many AWS locations from around the world are available. The full list is retrievable through the [Datadog API][2]. You can also set up a [Private Location][3] to run a Synthetics API test on a private endpoint not accessible from the public internet.
+6. **How often should Datadog run the test?** Set the interval for how often the test will run. Intervals are available between every one minute to once per week.
+7. Click on **Test URL** to try out the request configuration. You should see a response preview show up on the right side of your screen.
+
+[1]: /synthetics
+[2]: /api/?lang=bash#get-available-locations
+[3]: /synthetics/private_locations
+{{% /tab %}}
+
+{{% tab "SSL Test" %}}
+
+{{< img src="synthetics/api_tests/make-ssl-request.png" alt="Make SSL Request" responsive="true" style="width:80%;" >}}
+
+1. **Choose request type**: `SSL`
+2. Specify the `Host` and the SSL `Port`. By default, the port is set to _443_.
+3. **Name**: The name of your API test.
+4. **Select your tags**: Select the tags attached to your API test. Use the `<KEY>:<VALUE>` format to filter on a `<VALUE>` for a given `<KEY>` on the [Synthetics page][1].
+5. **Locations**: Select which Datadog managed locations to run the test from. Many AWS locations from around the world are available. The full list is retrievable through the [Datadog API][2]. You can also set up a [Private Location][3] to run a Synthetics API test on a private endpoint not accessible from the public internet.
+6. **How often should Datadog run the test?** Set the interval for how often the test will run. Intervals are available between every five minutes to once per week.
+7. Click on **Test Connection** to try out the request configuration. You should see a response preview show up on the right side of your screen.
+[1]: /synthetics
+[2]: /api/?lang=bash#get-available-locations
+[3]: /synthetics/private_locations
+{{% /tab %}}
+
+{{< /tabs >}}
+
+#### Set alert conditions and assertions
```

Comment:
Would put it the other way around: Set assertions & alert conditions (since assertions are mandatory VS alert conditions are optional)